lib/membrane_opus.ex

```diff
@@ -8,31 +8,31 @@ defmodule Membrane.Opus.Description do
   @typedoc """
   Number of channels transmitted in the stream.
   """
-  @type channels_t :: 1 | 2
+  @type channels_t :: 1 | 2 | nil
 
   @typedoc """
-  Enable/disable in-band _Forward Error Correction_ (FEC).
+  Determines whether the stream is equipped with in-band _Forward Error Correction_ (FEC).
   """
-  @type fec_enabled_t :: boolean
+  @type fec_available_t :: boolean | nil
```

Comment:
Is this actually needed? I haven't found a way to detect that FEC is enabled, so for RTP received streams we'll never know and when encoding it is enough to set it as encoder option